src/application/agora/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const TWITTER_HANDLE_TO_AGORA: Record<string, string> = {
+  optimism: 'optimism.eth',
+} as const;
```

Comment:
actually maybe you can change it in both snapshot and agora? I think it's fine to just leave Record and remove `as const` since we don't need to know exact keys/values, it's just a lookup obj

Reply:
Yeah, that makes sense 😄